addons/mail/static/src/scss/abstract_thread_window.scss

```diff
@@ -1,6 +1,7 @@
 $o-mail-thread-window-zindex: $zindex-modal + 1 !default;
 
 .o_thread_window {
+    direction: ltr;
```

Comment:
Yes, I agree that by default it is ```ltr``` but we added direction intentionally so that when rtlcss preprocessor runs then it can convert all ```ltr``` direction to ```rtl```